src/interbank/dto/query.dto.ts

```diff
@@ -0,0 +1,5 @@
+export class QueryDTO {
+  from: string;
+  to: string;
+  bankID: number;
```

Comment:
**question**: does FE know bank ID? Should we change it to `bank: 'HCMUSBank'`?

Reply:
We will provide FE an api to get a list of interbanks so they will have bankID.